libs/components/ag-grid/src/lib/styles/_ag-grid-extra.scss

```diff
@@ -0,0 +1,644 @@
+@use 'node_modules/ag-grid-community/styles' as ag;
+
+/**
+ * Additional styles based on https://github.com/ag-grid/ag-grid/blob/b32.0.2/community-modules/styles/src/internal/themes/quartz/_index.scss
```

Comment:
Do you have more context around this change?

Reply:
After re-reviewing these changes, I'm dropping them.